txndstransform/txndstransform.go

```diff
@@ -0,0 +1,209 @@
+package txndstransform
```

Comment:
This is a utility that I did in `go-threads`. It allows to Wrap a `TxnDatastore` with a prefix-namespace.
The existing one by PL only works with `datastore.Datastore` (not `TxnDatastore`).
Since we use badger, and I needed to use it as `TxnDatastore` (since I needed transactions) and also wanted to prefix each index with their own namespace... I just re-leveraged this work.

Reply:
Nice! Maybe this could be it's own lib at some point.